python/utils/trace.py

```diff
@@ -874,6 +874,125 @@ def configure_shimtile_dma_tracing_aie2(
     )
 
 
+def push_bd_to_task_queue(
+    shim, bd_id, direction, channel, enable_token  # 0: s2mm, 1: mm2s
+):
+    task_queue_addr = 0x0
+    if direction == 0:  # s2mm
```

Comment:
Can you use constant variable? The comment you added is nice, but reusable constants with descriptive names seem nicer (at least, to me) :)